schema/snapcraft.yaml

```diff
@@ -168,6 +168,15 @@ properties:
             enum:
               - simple
               - forking
+          restart-condition:
+              type: string
+              enum:
+                - never
```

Comment:
I grabbed them from here https://github.com/ubuntu-core/snappy/blob/master/docs/meta.md
So more research is in order